### PR TITLE
feat: Implemented data transfer between pages

### PR DESCRIPTION
**Pull request by AndrewKaenko**

### CODE REVIEW

File: src/containers/SearchResult.jsx [hunk 1]
```diff
@@ -85,18 +86,20 @@ const TimeP = styled.p`
...
-const BookBtn = styled.div`
+const BookBtn = styled(Link)`
   border: 3px solid #d30000;
   width: 150px;
   height: 35px;
   display: flex;
   align-items: center;
   justify-content: center;
   cursor: pointer;
+  text-decoration: none:
```
Comment (alexandrtovmach) [inline src/containers/SearchResult.jsx:97]:
```suggestion
  text-decoration: none;
```

File: src/containers/Form.jsx [hunk 2]
```diff
@@ -96,13 +97,15 @@ const Input = styled.input`
...
+const OrderForm = ({ onChangeForm, children }) => {
   const { register, handleSubmit } = useForm({ mode: "onChange" })
 
+  // eslint-disable-next-line prefer-const
+  let history = useHistory()
   return (
     <FormCapacity>
       <BlockEnvelop>
-        <PrevLink>&#10510;</PrevLink>
+        <PrevLink onClick={() => history.goBack()}>&#8592;{children}</PrevLink>
```
Comment (alexandrtovmach) [inline src/containers/Form.jsx:108]: why do you need `{children}` here?

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

**Comment by alexandrtovmach** [comment]
image
please fix styles of arrow, to make it aligned

**Comment by AndrewKaenko** [comment]
I understood.  Upon arrival I will do

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**